docs/get-started/getting-started-with-rapids-accelerator-on-databricks.md

```diff
@@ -0,0 +1,80 @@
+
+## Prerequisites
+* Apache Spark 3.0 running in DataBricks Runtime 7.0 ML with GPU 
+    * AWS: 7.0 ML (includes Apache Spark 3.0.0, GPU, Scala 2.12)
+    * Azure: 7.0 ML (GPU, Scala 2.12, Spark 3.0.0)
+
+The number of GPUs per node dictates the number of Spark executors that can run in that node. Each executor should only be allowed to run 1 task at any given time.
+
+## Start a Databricks Cluster
+Create a Databricks cluster by going to Clusters, then clicking “+ Create Cluster”. Ensure the cluster meets the prerequisites above by configuring it as follows:
+1. On AWS, make sure to use 7.0 ML (includes Apache Spark 3.0.0, GPU, Scala 2.12), or for Azure, choose 7.0 ML (GPU, Scala 2.12, Spark 3.0.0).
+2. Under Autopilot Options, disable auto scaling.
+3. Choose the number of workers that matches the number of GPUs you want to use.
+4. Select a worker type. On AWS, use nodes with 1 GPU each such as p3.xlarge or g4dn.xlarge. p2 nodes do not meet the architecture requirements for the Spark worker (although they can be used for the driver node). For Azure, choose GPU nodes such as Standard_NC6s_v3
+5. Select the driver type. Generally this can be set to be the same as the worker.
+
+## Advance Cluster Configuration
+
+We will need to create an initialization script for the cluster that installs the RAPIDS jars to the cluster.
+
+1. To create the initialization script, import the initialization script notebook from the repo [generate-init-script.ipynb](../demo/Databricks/) to your workspace. See [Managing Notebooks](https://docs.databricks.com/user-guide/notebooks/notebook-manage.html) on how to import a notebook, then open the notebook.
+2. Once you are in the notebook, click the “Run All” button.
+3. Ensure that the newly created init.sh script is present in the output from cell 2 and that the contents of the script are correct..
+4. Go back and edit your cluster to configure it to use the init script. To do this, click the “Clusters” button on the left panel, then select your cluster.
+5. Click the “Edit” button, then navigate down to the “Advanced Options” section.
+Select the “Init Scripts” tab in the advanced options section, and paste the initialization script: `dbfs:/databricks/init_scripts/init.sh`, then click “Add”
+
+![Init Script](../img/initscript.png)
+
+6. Now select the “Spark” tab, and paste the following config options into the Spark Config section:
+    ```bash
+    spark.plugins com.nvidia.spark.SQLPlugin
+    spark.sql.parquet.filterPushdown false
+    spark.rapids.sql.incompatibleOps.enabled true
+    spark.rapids.memory.pinnedPool.size 2G
+    spark.locality.wait 0s
+    spark.hadoop.mapreduce.fileoutputcommitter.algorithm.version 2
+    spark.executor.extraJavaOptions "-Dai.rapids.cudf.prefer-pinned=true"

```

Comment:
Could we add an explination for each of these configs so that users know what they are doing instead of just blindly copying configs that might do things that they don't want.  Like `spark.rapids.sql.incompatibleOps.enabled`

Reply:
For the short-term, we can point them to configs & Tuning guide for detailed explanation.